### PR TITLE
migrate management API calls to apiv2

### PR DESCRIPTION
**Pull request by bkendall**
### Description

Migrates all the API calls in the management folder to apiv2. Done by using `nock` in the tests (which validates the calls being made) and then replacing the actual calls, so I know the requests are the same.

### Scenarios Tested

- [x] Test making/removing/listing apps
- [x] Test working with a database

### CODE REVIEW

File: src/test/management/apps.spec.ts [hunk 1]
```diff
@@ -61,21 +64,22 @@ function generateWebAppList(counts: number): WebAppMetadata[] {
   }));
 }
 
-describe("App management", () => {
+describe.only("App management", () => {
```
Comment (joehan) [inline src/test/management/apps.spec.ts:67]: Forgot an only

**Review by joehan — CHANGES_REQUESTED** [review]
Left a .only in here

**Review by joehan — COMMENTED** [review]
Other than the .only, this looks good to me (once the 2 cases you listed are tested

File: src/appdistribution/client.ts [hunk 2]
```diff
@@ -196,7 +196,7 @@ export class AppDistributionClient {
         { emails: string[] },
         BatchRemoveTestersResponse
       >({
-        method: "POST",
+        method: "DELETE",
```
Comment (joehan) [inline src/appdistribution/client.ts:199]: Was this just broken before?

**Review by joehan — APPROVED** [review]

File: src/appdistribution/client.ts (hunk 2 above)
Comment (bkendall) [reply src/appdistribution/client.ts:199]: I changed the wrong place.

**Pull request merged**